pkgs/development/interpreters/nextflow/default.nix

```
@@ -0,0 +1,43 @@
+{ lib,
+  stdenv,
+  fetchurl,
+  makeWrapper,
+  jre,
+  wget,
+  which,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "nextflow";
+  version = "21.10.6";
+
+  src = fetchurl {
+    url = "https://github.com/nextflow-io/nextflow/releases/download/v${version}/nextflow-${version}-all";
+    sha256 = "0l9hi51vrhvfx3px2pxw7lp4h21n8ks50x4icfk3hbgl2hwf7fvx";
+  };
+
+  nativeBuildInputs = [makeWrapper];
+  buildInputs = [jre wget which];
```

For consistency with the rest of nixpkgs:
```suggestion
  nativeBuildInputs = [ makeWrapper ];
  buildInputs = [ jre wget which ];
```
btw, you can automatically do trivial formatting stuff with `nixpkgs-fmt`.